### PR TITLE
Wallet issuance to return tx instead of tx id

### PR DESCRIPTION
**Pull request by felipemadero**
## Why this should be merged
Currently txId is returned by wallet issuance functions, giving the clients less information than available. Some users may want the tx, and the txId can be obtained from it.

## How this works
Returns tx instead of txId for all applicable function

## How this was tested
unit testing

### CODE REVIEW

File: wallet/chain/p/wallet.go [hunk 1]
```diff
@@ -451,59 +451,64 @@ func (w *wallet) IssueAddPermissionlessDelegatorTx(
...
+		return nil, err
 	}
 
-	return w.IssueTx(tx, options...)
+	err = w.IssueTx(tx, options...)
+	if err != nil {
+		return nil, err
+	}
+
+	return tx, err
```
Comment (StephenButtolph) [inline wallet/chain/p/wallet.go:483]:
```suggestion
	return tx, w.IssueTx(tx, options...)
```

File: wallet/chain/x/wallet.go [hunk 2]
```diff
@@ -272,54 +272,59 @@ func (w *wallet) IssueExportTx(
...
+		return nil, err
 	}
 
-	return w.IssueTx(tx, options...)
+	err = w.IssueTx(tx, options...)
+	if err != nil {
+		return nil, err
+	}
+
+	return tx, nil
```
Comment (StephenButtolph) [inline wallet/chain/x/wallet.go:299]:
```suggestion
	return tx, w.IssueTx(tx, options...)
```

File: wallet/subnet/primary/example_test.go
Comment (StephenButtolph) [inline wallet/subnet/primary/example_test.go:1]: There are more txID variables that need to be updated in this file

File: wallet/subnet/primary/examples/create-asset/main.go
Comment (StephenButtolph) [inline wallet/subnet/primary/examples/create-asset/main.go:1]: I think all the other examples need to be updated as well

**Review by StephenButtolph — COMMENTED** [review]
I think linting is failing

**Review by abi87 — APPROVED** [review]
change seems ok to me. I just don't understand what extra information client may require, apart from the txID. @felipemadero can you clarify?

**Comment by StephenButtolph** [comment]
> I just don't understand what extra information client may require, apart from the txID.

Generally I think it makes sense to provide the client exactly what was issued. We already have the information locally - so we can avoid an RPC call if they need to lookup the Tx.

**Review by StephenButtolph — APPROVED** [review]

**Pull request merged**